speedy-recovery-client/src/components/secured/patientinformation/Practitioner/PractitionerCarePlanPane.js

```diff
@@ -0,0 +1,113 @@
+            if (carePlansNum) {
+                const header = [
+                    <Table.Row key={"carePlanRow"}>
+                        <Table.HeaderCell id="parentTableCell">#</Table.HeaderCell>
+                        <Table.HeaderCell id="parentTableCell">Category</Table.HeaderCell>
+                        <Table.HeaderCell id="parentTableCell">Activities</Table.HeaderCell>
+                        <Table.HeaderCell id="parentTableCell">Status</Table.HeaderCell>
+                        <Table.HeaderCell id="parentTableCell">Period</Table.HeaderCell>
+                    </Table.Row>
+                ];
+                table.push(
+                    <Table.Header key="carePlanTableHeader" id="parentTableHeader">
+                        {header}
+                    </Table.Header>
+                );
+            }
+            const body = [];
+            for (let i = 0; i < carePlansNum; i++) {
+                const children = [];
+
+                children.push(
+                    <Table.Cell key={"carePlanOrderCell" + i}>
+                        <Label color="blue" ribbon>
+                            {`${i + 1}`}{" "}
+                        </Label>
+                    </Table.Cell>
+                );
+                children.push(
+                    <Table.Cell key={"carePlanCategoryCell" + i} id="parentTableCell">
+                        {<h4>{carePlans[i].category}</h4>}
+                    </Table.Cell>
+                );
+                children.push(
+                    <Table.Cell key={"carePlanActCell" + i} id="parentTableCell">
+                        {<h4>{carePlans[i].activities}</h4>}
+                    </Table.Cell>
+                );
+                children.push(
+                    <Table.Cell key={"carePlanStatusCell" + i} id="parentTableCell">
+                        {<h4>{carePlans[i].status}</h4>}
+                    </Table.Cell>
+                );
+                children.push(
+                    <Table.Cell key={"carePlanPeriodCell" + i} id="parentTableCell">
+                        {<h4>{carePlans[i].period}</h4>}
+                    </Table.Cell>
+                );
+
+                body.push(
+                    <Table.Row key={"carePlanRow2" + i} id="parentTableRow">
+                        {children}
+                    </Table.Row>
+                );
+            }
+            table.push(
+                <Table.Body key="carePlanTableBody" id="parentTableBody">
+                    {body}
+                </Table.Body>
+            );
+            const carePlanTable = [];
+            carePlanTable.push(
+                <Table key="carePlanTable" color="blue">
+                    {table}
+                </Table>
+            );
+            return carePlanTable;
+        };
+
+        return (
+            <div>
+                <Tab.Pane>
+                    <h4>
+                        You have {carePlansNum} recorded care plan.
```

Comment:
instead of this line you can also say "You have {carePlansNum} recorded care plan for {patientName}". Otherwise it looks like you are showing the practitioner's record.

Reply:
same for all the other records